changelog.txt

```diff
@@ -1,3 +1,11 @@
+Version 1.20.0
```

Comment:
bad merge? These are not part of this PR?

Reply:
Not a bad merge, just a lack of a one: this PR is pointing to the "nimbus-removal" feature branch, and two other PRs got merged into that feature branch before this PR.

I just pulled current feature branch into this PR's branch, and now it seems to have the correct file list again.